src/App.vue

```diff
@@ -31,7 +34,15 @@ export default {
   '$route.meta'({hideNavigation}){
     this.showNav = !hideNavigation;
   }
- }
+  },
+  created() {
+    //Pass store to window object on testing
+    if (window.Cypress !== null) { 
+    window.gameStore = useGameStore();
+    window.authStore = useAuthStore();
+    window.gameListStore = useGameListStore();
+    }
```

Comment:
I'm wondering if it would be better to do 
```
window.store = { 
 gameStore : useGameStore(),
 authStore : useAuthStore(),
 gameListStore: useGameListStore(),
 };

Reply:
Interesting idea. Ultimately it doesn't matter much either way, but I think it's more in the spirit of pinia to keep it flat like you already have it since that's closer to how you'd grab them with script setup. And really, what's the point in an additional layer of property access?

Reply:
Cool, my thought was maybe putting it inside window.store would keep them all together, but you right it doesn't really matter. :)

Reply:
It might still be better to keep these in the cuttle namespace regardless of the store nesting, right now you're polluting the window object.